modules/admin_manual/pages/enterprise/external_storage/windows-network-drive_configuration.adoc

```diff
@@ -25,16 +24,30 @@ Any Windows file share and Samba servers on Linux and other Unix-type operating
 file-sharing protocol. The files and directories on the SMB/CIFS server will be visible on your Files page
 just like your other ownCloud files and folders.
 
-They are labeled with a little four-pane Windows-style icon, and the left pane of your Files page includes
-a Windows Network Drive filter. Figure 1 shows a new Windows Network Drive share marked with red warnings.
+Compared to standard SMB access, WND has advanced features like:
 
-These indicate that ownCloud cannot connect to the share because it requires the user to login, it is not
-available, or there is an error in the configuration.
+* User lockout prevention and password reset
+* More authentication mechanisms against the backend
+* Change notifications
+* Enhanced ACL support
 
+Here is a brief description of the advanced features:
+
+Depending of the Windows or Samba policy, the user could get locked out of his account if he enters a wrong password a number of times. The lockout prevention tries to avoid this from happening, by resetting the password if it is wrong. For the case of the ownCloud's standard SMB connector, the password won't be reset. It could happen, that the user gets locked out of the file server.
```

Comment:
```suggestion
Depending on the Windows or Samba policy, the user could get locked out of his account if he enters a wrong password a number of times. The lockout prevention tries to avoid this from happening, by resetting the password if it is wrong. For the case of ownCloud's standard SMB connector, the password won't be reset. It could happen, that the user gets locked out of the file server.
```